src/schemas-tests/pa-yaml/v3.0/Examples/Src/DataSources/DataSources1.pa.yaml

```diff
@@ -1,10 +1,13 @@
 DataSources:
```

Comment:
not required for this PR, but do updatie the yaml schema file here to support native intellisense and validation support within VS Code. I can help you get started with this too. I've already added a simple validation for the DataSources top-level property.

see: https://github.com/microsoft/PowerApps-Tooling/blob/master/src/schemas/pa-yaml/v3.0/pa.schema.yaml